src/pal/inc/pal.h

```diff
@@ -5222,6 +5222,22 @@ DWORD
 PALAPI
 EnsureOpenSslInitialized();
 
+PALIMPORT
+DWORD
+PALAPI
+ForkAndExecProcess(
+           IN LPCSTR lpFilename,
+           IN LPVOID lpArgv,
+           IN LPVOID lpEnvp,
```

Comment:
We should use standard C++ type names in the signatures of corefx helpers: char*, int, ... .

bool is a tricky one. I would use int with flag bits instead.